pkg/minikube/constants/constants_linux.go

```diff
@@ -18,7 +18,14 @@ limitations under the License.
 
 package constants
 
+import (
+	"k8s.io/client-go/util/homedir"
+)
+
 var SupportedVMDrivers = [...]string{
 	"virtualbox",
 	"kvm",
 }
+
+var DefaultMountDir = homedir.HomeDir()
```

Comment:
Looked into it a little more.  Looks like `os/user` doesn't return the homedir for windows.  https://golang.org/src/os/user/lookup_windows.go#L127

I would just leave this as is for now then